docs/src/main/asciidoc/compose-dev-services.adoc

```diff
@@ -0,0 +1,632 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Compose Dev Services
+include::_attributes.adoc[]
+:categories: core
+:summary: Configure custom Dev Services using Docker or Podman Compose.
+:topics: dev-services,dev-mode,testing,compose
+
+xref:dev-services.adoc[Quarkus Dev Services] automatically provisions unconfigured services in development and test mode.
```

Comment:
this intro is not really informative - at least to me it doesn't explain much what it enables.

i.e. what does "automatically provision unconfigured" services?

can we make it a bit more about what it enables users to do?

Reply:
This "automatically provision unconfigured" was taken from the devservices guide. But maybe it doesn't belong in the introduction.